Cython/Utility/Builtins.c

```diff
@@ -344,6 +344,51 @@ static CYTHON_INLINE PyObject* __Pyx_divmod_int(int a, int b) {
 }
 
 
+//////////////////// divmod_long.proto //////////////////
+
+static CYTHON_INLINE PyObject* __Pyx_divmod_long(long a, long b); /*proto*/
+
+
+//////////////////// divmod_long //////////////////
+
+static CYTHON_INLINE PyObject* __Pyx_divmod_long(long a, long b) {
```

Comment:
Since the code is practically the same as above, except for the types, I think we can use Tempita template generation to avoid the duplication. Look for `TempitaUtilityCode`. You can then easy add `long long` as well.

Reply:
I will have a look on the Tempita template later.
Before submitting this PR, I was more hesitate on how to declare `long long` in the `Cython/Compiler/Builtin.py` file.